wlauto/common/android/workload.py

```diff
@@ -84,7 +84,7 @@ def __init__(self, device, _call_super=True, **kwargs):  # pylint: disable=W0613
         self.uiauto_params = ParameterDict()
 
     def init_resources(self, context):
-        self.uiauto_file = context.resolver.get(wlauto.common.android.resources.uiautoApkFile(self))
+        self.uiauto_file = context.resolver.get(wlauto.common.android.resources.ApkFile(self, uiauto_apk=True))
```

Comment:
Rather than using the fully-qualified name here, import `ApkFile` into the current namespace.